docs/details/api.md

```diff
@@ -19,3 +19,45 @@ The objects visible are nodes, services, and agents:
 Nodes, Services, and Agents have **Types** which define specific their properties, and their specific logic.
 
 Nodes and Services are external by nature – we do not manage them (create, destroy), but merely maintain a list of them (add to inventory, remove from inventory) in `pmm-managed`. Most Agents are started and stopped by `pmm-agent`. One exception is the External Exporter Type which is started externally.
+
+
+## API Keys and authentication
+
+API keys are used to control access to the PMM server components and resources. With an API key, you are authenticated to the PMM server, have access to PMM server components and resources, and perform various actions on them. You can use API keys as a replacement for basic authentication.
+
+### Generate API keys
+
+PMM uses the Grafana API keys for authentication. Following are the steps to generate the API keys:
+
+1. Login to PMM.
+2. From the side menu, click *Configuration  → API keys*.
+3. On the Configuration page, click *Add API Key*.
+4. *Add API key* dialog box opens.
+5. Enter  the following to generate an API key:
+    - key name (you can give any desired name)
+    - Select the Role from the dropdown 
+    - Enter a value in the Time to live text box (hover on the tooltip for more information). 
+    - Click Add.
+
+6. *API Key Created* window displays your newly created key. Make sure to copy your key and keep it secure. 
+
+### Authenticate
+
+You can authenticate your request using the HTTP header.
+
+**Example**
+
+```sh
+curl -H "Authorization: Bearer <api_key> http://127.0.0.1/graph/api/dashboards/home
+```
+
+### Use an API key
+
+You can pass the API key into a REST API call as a query parameter in the following format. Replace `api_key` with your API key.
+
+**Example**
+
+
+```sh
+curl -X GET https://api_key:API_KEY@localhost/v1/version
```

Comment:
same here, we need `-k` option:
```
[dkondratenko@denlen tmp]$ curl -X GET https://api_key:eyJrIjoiYVcyQWlSZ3dMaHhIS09ydHRzTlV0SVB2dERwckozNkIiLCJuIjoicG1tIiwiaWQiOjF9@192.168.49.2:30443/v1/version
curl: (60) SSL certificate problem: self signed certificate
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
[dkondratenko@denlen tmp]$ curl -k -X GET https://api_key:eyJrIjoiYVcyQWlSZ3dMaHhIS09ydHRzTlV0SVB2dERwckozNkIiLCJuIjoicG1tIiwiaWQiOjF9@192.168.49.2:30443/v1/version
{
  "version": "2.25.0",
  "server": {
    "version": "2.25.0",
    "full_version": "2.25.0-64.2112130941.7f24595.el7",
    "timestamp": "2021-12-13T09:41:40Z"
  },
  "managed": {
    "version": "2.25.0",
    "full_version": "dc091abb30f1e0ac8769a02d0587aa1be653a58c",
    "timestamp": "2021-12-13T09:39:25Z"
  },
  "distribution_method": "DOCKER"```

Reply:
@rnovikovP what should we do - ask ppl to use insecure option or provide good instructions but fix cert issues?

Reply:
or should we have a section about how to install our certificates ? and mention it here?

Reply:
httpS! You right.
And yes, some notes about how to ignore certificates. 

We do the same on `pmm-admin config ...` and providing instructions for Certificates is out of the scope of this PR